packages/aws-cdk/lib/commands/migrate.ts

```diff
@@ -79,7 +79,7 @@ export async function generateCdkApp(stackName: string, stack: string, language:
  */
 export function generateStack(template: string, stackName: string, language: string) {
   try {
-    const formattedStackName = `${camelCase(decamelize(stackName), { pascalCase: true })}Stack`;
+    const formattedStackName = `${camelCase(decamelize(stackName), { pascalCase: true })}` + language == 'csharp' ? '' : 'Stack';
```

Comment:
This breaks all the csharp test. Can you elaborate on what you were trying to fix here?

Reply:
Typescript fun fact of the day. "+" has a higher priority than "?" so this evaluates in the incorrect order. Added some () to fix the PEMDAS here. This is only appending stack to the end if it is a language other than CSharp. This is because the CSharp standard is to name the namespace "appName" and then the stack "appNameStack". So since we want to maintain this then we need to make sure we are not appending Stack all the time to CSharp. If we didn't do this then we'd be naming things "myAppStack" and "myAppStackStack" which is a bit silly. Or in this case "migrateStackStack" and "migrateStackStackStack" which is even sillier.

All good now. Should be 5/5 once the cdk-from-cfn change is merged

```
Test Suites: 1 failed, 1 skipped, 1 of 2 total
Tests:       1 failed, 73 skipped, 4 passed, 78 total
Snapshots:   0 total
Time:        468.022 s
```